ivadomed/transforms.py

```diff
@@ -302,7 +304,6 @@ def __getitem__(self, item):
 class Crop(ImedTransform):
     def __init__(self, size):
         self.size = size if len(size) == 3 else size + [0]
-        self.is_2D = True if len(size) == 2 else False
```

Comment:
The reason why I check the 2D versus 3D here instead of via sample --> was to be robut to 3D data with shape[2] == 1 (yes I know.. not common). While by checking with the crop_size --> we are certain that the user wants a 2D transform.
What do you think @andreanne-lemay ?

Reply:
Yes I wanted to discuss that as well! The way it was before worked fine, why I changed it is so the transform automatically adapts to 2D or 3D output without having to change the tranforms. For instance, in the test_orientation.py I can use the same transforms for 2D and 3D. Also, I found it useful to not have to change multiple parameters when I want to train in 2D or 3D (now I only have to change the unet_3D bool).

Reply:
It is convenient indeed. Ok, we can go with that.